pkg/services/control/service.proto

```diff
@@ -167,6 +170,34 @@ message ListShardsResponse {
     Signature signature = 2;
 }
 
+// Request to list all objects in the node.
+message ListObjectsRequest {
+    // Request body structure.
+    message Body {
+    }
+
+    // Body of the request message.
+    Body body = 1;
+
+    // Body signature.
+    Signature signature = 2;
+}
+
+// ListObjects response.
+message ListObjectsResponse {
+    // Response body structure.
+    message Body {
+        // Address of the object.
+        bytes object_address = 1;
```

Comment:
per-object message would be very inefficient, especially with each message signing. Suggest to make it `repeated`. WIth this, element limit should be contolled. See similar [implementation](https://github.com/nspcc-dev/neofs-node/blob/0a6e88a30c8e07605cd151cbabada1818b2e9e8f/pkg/services/object/transport_splitter.go#L151)

Reply:
Done with repeated field.